packages/web-pkg/src/composables/driveResolver/useDriveResolver.ts

```diff
@@ -16,7 +16,13 @@ interface DriveResolverOptions {
   driveAliasAndItem?: Ref<string>
 }
 
-export const useDriveResolver = (options: DriveResolverOptions = {}) => {
+interface ResolvedDrive {
```

Comment:
I'd prefer
```suggestion
interface DriveResolverResult {
```
as we've already used that naming scheme a lot (`...Options` and `...Result`).